### ISSUE TITLE
function stepBubbles() missing from code

### ISSUE THREAD

**Issue by john-ring** [Venturocket/angular-slider#72]
In the template, the span for the step indicators is:
![screen shot 2014-10-24 at 2 02 46 pm](https://cloud.githubusercontent.com/assets/5882209/4773936/03be992e-5ba8-11e4-91eb-94aa0f76a1e2.png)
However the function stepBubbles() does not exist anywhere in the code, so you never get any tick indicators besides the start and end ones.

**Comment by drgould** [comment]
Yes, that's something I was wanting to implement but never had the time. I'll make sure this is implemented in the rewrite.

**Comment by lcrojanouninorte** [comment]
how can i implement that
